docs/sources/visualizations/dashboards/build-dashboards/annotate-visualizations/index.md

```diff
@@ -149,7 +149,10 @@ To add a new annotation query to a dashboard, follow these steps:
    You can also click **Open advanced data source picker** to see more options, including adding a data source (Admins only).
 
 1. If you don't want to use the annotation query right away, clear the **Enabled** checkbox.
-1. If you don't want the annotation query toggle to be displayed in the dashboard, select the **Hidden** checkbox.
+1. Choose a **Control display** option:
```

Comment:
The bolded text should be the same as the UI label. So assuming the image in the PR description is correct, I'm making this suggestion.
```suggestion
1. Select one of the following options in the **Show annotation controls in** drop-down list to control where annotations are displayed:
```